apps/mark/frontend/src/app_quit_on_idle.test.tsx

```diff
@@ -64,3 +70,46 @@ test('Insert Card screen idle timeout to quit app', async () => {
   await advanceTimersAndPromises(QUIT_KIOSK_IDLE_SECONDS);
   expect(window.kiosk?.quit).toHaveBeenCalledTimes(1);
 });
+
+test('Voter idle timeout', async () => {
```

Comment:
Had to add this test to get back up to 100% coverage since this timeout was previously tested in a file specific to voter card flows